src/main/java/dk/kb/ds/cumulus/export/Configuration.java

```diff
@@ -115,11 +122,15 @@ public static String getCollection(){
     public static String getOutputFile(){
         return instance().outputFile;
     }
-
     public static String getType(){
         return instance().type;
     }
-
+    public static String getLimited(){
+        return instance().limited;
+    }
+    public static String getCounter(){
+        return instance().counter;
+    }
```

Comment:
While we are cleaning up: The key names are not very descriptive and I find it cumbersome to have 2 keys controlling the amount of output. Convention has it that `-1` means "no limit", so how about switching to a single key called `maxrecords`, `recordlimit` or something like that?